docs/developers/index.md

```diff
@@ -32,6 +32,15 @@ Need a more basic primer first? Check out [ethereum.org/learn.](/learn/)
 
 Any program that runs on the Ethereum Virtual Machine (EVM) is commonly referred to as a “smart contract”. The most popular languages for writing smart contracts on Ethereum are **Solidity** and **Vyper**, though there are [others under development](https://github.com/ConsenSys/ethereum-developer-tools-list#smart-contract-languages).
 
+## Language Specific Landing Pages
```

Comment:
1. Could we change "Landing Pages" to "Resources"?
2. I don't think this is the best location for this section. The Solidity/Vyper/other option subheaders below are intended to fit under the "Smart Contract Languages" section. Can we please move this section below those subheaders?

Reply:
Fixed!